### DISCUSSION TITLE
基建-----怎么实现让医疗小车自动套的小车保持0理智的同时在宿舍使用自动填充？可以在Json文件里面加Exemption吗？

### DISCUSSION THREAD

**Discussion by RadanPY** [MaaAssistantArknights/MaaAssistantArknights#9494] [General]
rt

**Comment by ABA2396** [comment]
塞进加工站

**Comment by Constrat** [comment]
Either put it in 加工站 as ABA said, or remove the autofill and create a json, learning how to put the various operators. It's a hassle I know, but in the long term it will be beneficial having your own base json, and disabling auto fill.

For complex rotations that's the only way (for example Rhine Lab Muelsyse 25% powerplant)
